moveit_core/trajectory_processing/include/moveit/trajectory_processing/ruckig_traj_smoothing.h

```diff
@@ -124,16 +132,18 @@ class RuckigSmoothing
    */
   static void initializeRuckigState(const moveit::core::RobotState& first_waypoint,
                                     const moveit::core::JointModelGroup* joint_group,
-                                    ruckig::InputParameter<ruckig::DynamicDOFs>& ruckig_input,
-                                    ruckig::OutputParameter<ruckig::DynamicDOFs>& ruckig_output);
```

Comment:
You need to remove this from the API documentation too.

Reply:
done